flag-types.json

```diff
@@ -80,12 +96,16 @@
   {
     "name": "Uint64",
     "type": "uint64",
+    "value": true,
+    "dest": true,
     "context_default": "0",
     "parser": "strconv.ParseUint(f.Value.String(), 0, 64)"
   },
   {
     "name": "Uint",
     "type": "uint",
+    "value": true,
+    "dest": true,
```

Comment:
wow I have no idea what this file is even for 🤔

Reply:
This file is the source that is parsed to generate the flag types. The `FlagType` struct corresponds to this.